web/profiles/custom/os2loop/modules/os2loop_documents/src/EventSubscriber/EntityPrintEventSubscriber.php

```diff
@@ -2,56 +2,103 @@
 
 namespace Drupal\os2loop_documents\EventSubscriber;
 
+use Drupal\Component\Utility\Html;
 use Drupal\Core\Url;
 use Drupal\entity_print\Event\PreSendPrintEvent;
 use Drupal\entity_print\Event\PrintEvents;
-use Drupal\entity_print\Plugin\EntityPrint\PrintEngine\PhpWkhtmlToPdf;
+use Drupal\entity_print\Event\PrintHtmlAlterEvent;
+use Drupal\entity_print\Plugin\EntityPrint\PrintEngine\DomPdf;
 use Drupal\node\NodeInterface;
 use Drupal\os2loop_documents\Helper\NodeHelper;
+use Masterminds\HTML5;
+use Drupal\Core\Site\Settings;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
 
 /**
  * Event subscriber.
  */
 class EntityPrintEventSubscriber implements EventSubscriberInterface {
+  /**
+   * The request stack.
+   *
+   * @var \Symfony\Component\HttpFoundation\RequestStack
+   */
+  protected $requestStack;
+
+  /**
+   * EntityPrintEventSubscriber constructor.
+   *
+   * @param \Symfony\Component\HttpFoundation\RequestStack $request_stack
+   *   The request stack.
+   */
+  public function __construct(RequestStack $request_stack) {
```

Comment:
Use https://stitcher.io/blog/constructor-promotion-in-php-8 here and make the property private.

Reply:
Changed